src/components/contacts/index.tsx

```diff
@@ -0,0 +1,20 @@
+import Table from '../common/table';
+import { columns, sortingOptions } from './helper';
+import { contacts, total } from '../../store/contacts/selector';
+import { deleteContactAsync, fetchContactAsync } from '../../store/contacts/thunk';
+
+const ContactsTable = () => {
+  return (
+    <Table
+      columns={columns}
+      fetch={fetchContactAsync}
+      items={contacts}
+      total={total}
+      onDelete={deleteContactAsync}
+      sortingOptions={sortingOptions}
+      disabledFilter
+      contactsForm
```

Comment:
lets replace it with formType which will use Enum like:
enum EFromType {
Contacts = "Contacts",
Tickets = "Tickets"
}
Then you can use this types everywhere you want

Reply:
fix: remade with Enum